block/submit.go

```diff
@@ -201,15 +201,21 @@ func (m *Manager) CreateBatch(maxBatchSize uint64, startHeight uint64, endHeight
 			return nil, fmt.Errorf("load commit: h: %d: %w", h, err)
 		}
 
+		drsVersion, err := m.Store.LoadDRSVersion(block.Header.Height)
```

Comment:
I think this is not backward compatible.. we should open an issue on making it backward compatible (don't have to fix it now)

Reply:
https://github.com/dymensionxyz/dymint/issues/1170